bastion.cfhighlander.rb

```diff
@@ -17,7 +17,7 @@
       attribute 'DnsDomain'
     end
     subnet_parameters({'public'=>{'name'=>'Public'}}, maximum_availability_zones)
-    OutputParam component: 'vpc', name: "VPCId"
```

Comment:
@Guslington All of the `OutputParam` statements can be now replaced with `ComponentParam`. Parameters are autowired with output values by name. (or propagated to outer component if there is no component exposing such output).